### DISCUSSION TITLE
Run the simulation in Ubuntu 22.04.2 ARM64

### DISCUSSION THREAD

**Discussion by santysangro** [FS-Online/Driverless-Competition-Simulator#362] [Q&A]
Hi!
So I am currently trying to download the simulator in Ubuntu 22.04.2 ARM64, and I cannot seem to make it work. Can someone give me a little step by step on how to make it work? It would be greatly appreciated.
Thanks!

**Comment by RoyAmoyal** [comment]
Hey do you still need it? I can try to package the project to Linux ARM64
